### PR TITLE
Improvement, shoot, worker group node max / min count

### LINKED ISSUE #45: Expose min/max node count of specific worker groups
**What would you like to be added**:

We currently only expose the total min/max nodes of a shoot. In addition to this the min/max nodes of each worker group should be exposed.

### PR DESCRIPTION
**Pull request by Sinscerly**
Feature add worker node max / min count (and worker type as label)

**What this PR does / why we need it**:
- This PR adds two extra metrics: `garden_shoot_worker_node_max_total` and `garden_shoot_worker_node_min_total`. This gives insight about the min and max worker nodes for a shoot.

**Which issue(s) this PR fixes**:
Fixes  #45

**Special notes for your reviewer**:
Adding `garden_shoot_worker_node_max_total` and `garden_shoot_worker_node_min_total` can remove the current `garden_shoot_node_max_total` and `garden_shoot_node_min_total`. As the worker nodes counts can be summed for the total shoot.

**Release note**:
```
Adds: garden_shoot_worker_node_max_total and garden_shoot_worker_node_min_total. Also gives insight into the worker type (for example for OpenStack the Flavor name).
```

### CODE REVIEW

**Comment by gardener-robot-ci-3** [comment]
Thank you @Sinscerly for your contribution. Before I can start building your PR, a member of the organization must set the required label(s) {'reviewed/ok-to-test'}. Once started, you can check the build status in the PR checks section below.

**Review by rickardsjp — CHANGES_REQUESTED** [review]
Thank you @Sinscerly for your contribution. Only a small nitpick.

**Review by rickardsjp — APPROVED** [review]
/lgtm

File: README.md [hunk 1]
```diff
@@ -14,6 +14,8 @@ The `gardener-metrics-exporter` is a [Prometheus][] metrics exporter for
 | garden_shoot_condition                  | Condition state of a Shoot                                                | Shoot    | Gauge   |
 | garden_shoot_node_min_total             | Min node count of a Shoot                                                 | Shoot    | Gauge   |
 | garden_shoot_node_max_total             | Max node count of a Shoot                                                 | Shoot    | Gauge   |
+| garden_shoot_worker_node_min_total      | Min worker node count of a Shoot group                                    | Shoot    | Gauge   |
+| garden_shoot_worker_node_max_total      | Max worker node count of a Shoot worker group                                    | Shoot    | Gauge   |
```
[the remark this answers is not in the document]
Comment (Sinscerly) [reply README.md]: @rickardsjp thank you for pointing out. I've changed it to match the other descriptions:

```markdown
| garden_shoot_node_min_total             | Min node count of a Shoot                                                 | Shoot    | Gauge   |
| garden_shoot_node_max_total             | Max node count of a Shoot                                                 | Shoot    | Gauge   |
| garden_shoot_worker_node_min_total      | Min node count of a Shoot worker group                                    | Shoot    | Gauge   |
| garden_shoot_worker_node_max_total      | Max node count of a Shoot worker group                                    | Shoot    | Gauge   |
```
Let me know if this is what you meant.

[the remark this answers is not in the document]
Comment (rickardsjp) [reply README.md]: That's what I was referring to, thanks for taking care of it.

**Pull request merged**